### PR TITLE
Allow developers to add `CALayer` instances as POI 

### PR DESCRIPTION
**Pull request by yfujiki**
### Background

Allowing developers to add `CALayer` instances as POI (`LocationAnnotationNode`)
Current implementation with `UIView` does not allow dynamic change of contents because it is just taking a snapshot of the `UIView` when it was added. 

### Breaking Changes

None that I know of. 

### Meta
N/A

### PR Checklist
- [x] CI runs clean?
- [x] Appropriate label has been added to this PR (i.e., Bug, Enhancement, etc.).
- [x] Documentation has been added to all `open`, and `public` scoped methods and properties.
- [x] Changelog has been updated
- [ ] Tests have been added to all new features. (not a requirement, but helpful)
- [x] Image/GIFs have been added for all UI related changed.

### Screenshots

The current distance in this picture is updated live as you change your location for example.

CALayerPOI

### CODE REVIEW

File: Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift [hunk 1]
```diff
@@ -44,6 +44,24 @@ open class LocationAnnotationNode: LocationNode {
         self.init(location: location, image: view.image)
     }
 
+    public init(location: CLLocation?, layer: CALayer) {
+        let plane = SCNPlane(width: layer.bounds.size.width / 100, height: layer.bounds.size.height / 100)
+        plane.firstMaterial!.diffuse.contents = layer
```
Comment (intere) [inline Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift]: Don't use force unwraps, please

**Comment by intere** [comment]
I'll have to do some testing before I merge.  There was an issue with the original implementation of the `LocationAnnotationNode` initialize that took a view and the view hierarchy after popping the VC off of the NavigationController stack.  I'll need to make sure that's not still an issue with CALayers.  Thanks for the PR and for updating both README files!

File: Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift (hunk 1 above)
Comment (yfujiki) [reply Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift]: I actually followed exactly the same as the initialiser for the `UIImage` ;) (L18~)

Yup, definitely I will avoid force unwrap, and I will think about how we can avoid duplicate coding here.

Comment (intere) [reply Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift]: Thanks! I haven't had a chance to clean up all the force unwraps yet...  😞

**Comment by Pilot-Marc** [comment]
@yfujiki I've been testing this branch, and I really like this feature.  I've been able to modify the contents of the CALayer, but have not been successful at changing the size of the CALayer Billboard node.  Have you done this?

**Comment by yfujiki** [comment]
@Pilot-Marc Thank you for commenting (and sorry for leaving this branch for long...)

As for the `CALayer` size, I just set the `frame` and that worked. Only tricky part for me was that I need to convert the coordinate from CoreAnimation world (y-0 = bottom) to UIKit world (y-0 = top).

This is the snippet from my CALayer subclass :

```
    override init() {
        super.init()
        frame = CGRect(origin: .zero, size: CGSize(width: 261, height: 149))
        backgroundColor = UIColor.gray.cgColor
        cornerRadius = 10
        masksToBounds = true

        addSublayer(titleLayer)
        // ... adding different sublayers
```

File: Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift (hunk 1 above)
Comment (intere) [reply Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift]: you can literally just replace `!` with `?` here

**Comment by yfujiki** [comment]
@intere Sorry to disappear for long. Updated the PR as your comment. Hopefully this will do!

**Pull request merged**